### PR TITLE
Use canonical type name

### PR DESCRIPTION
**Pull request by fulldecent**
quick fix

### CODE REVIEW

File: EIPS/eip-2612.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ While it may be tempting to introduce `*_by_signature` counterparts for every ER
 Three new functions are added to the ERC-20 ABI:
 ```sol
 function permit(address owner, address spender, uint value, uint deadline, uint8 v, bytes32 r, bytes32 s) external
-function nonces(address owner) external view returns (uint)
+function nonces(address owner) external view returns (uint256)
```
Comment (xinbenlv) [inline EIPS/eip-2612.md:41]: While I am in support of spelling out it. the uint is an alias of uint256 for now. if you like to change line 41, consider also change everywhere else e.g. line 40, for consistency

**Comment by Skylerspencer** [comment]
Pull

**Comment by poxelman** [comment]
Go to rep0sitoris

**Comment by poxelman** [comment]
Create repositoris

File: EIPS/eip-2612.md (hunk 1 above)
Comment (poxelman) [reply EIPS/eip-2612.md:41]: Leave to riply

**Review by poxelman — DISMISSED** [review]
Create reapositori

**Pull request closed without merging**